### DISCUSSION TITLE
Not clear why vtherm in security mode

### DISCUSSION THREAD

**Discussion by flg** [jmcollin78/versatile_thermostat#668] [General]
Hello,

One of my vtherm (over valve) is in security mode with the following attributes read at ~14:48 (UTC+1):

```
security_delay_min: 60
security_min_on_percent: 0.5
security_default_on_percent: 0.1
last_temperature_datetime: 2024-11-27T13:59:25.670231+01:00
last_ext_temperature_datetime: 2024-11-27T14:13:30.549393+01:00
security_state: true
```

Last temperature and external temperature update were both less than `security_delay_min` ago so it's not clear to me what the security state is true. Does anybody has a hint?

Cheers

**Comment by jmcollin78** [comment]
Hello, have you check the link in this post ? 

https://github.com/jmcollin78/versatile_thermostat/discussions/261

**Reply by flg** [reply]
Yes I have. I'm sure I'm missing something but that's not obvious what as the last temperature readings happened within the last `security_delay_min`.

**Reply by jmcollin78** [reply]
ok. I need the attributes as explained here: https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#custom-attributes when the issue is running and if you can the logs when this happens.

I admit this is curious. 

Have you configured a lastSeen sensor ?
Have you disable the safety mode for the external temp sensor as suggested in readme ?

**Reply by flg** [reply]
There's no last seen sensor. I implemented your suggestion, i.e.:

```
versatile_thermostat:
  safety_mode:
    check_outdoor_sensor: false
```

The problem hasn't shown up since.

Thanks.
